PHPStan Level 1 issues

There are a few remaining issues to reach PHPStan level 1:

> ------ -------------------------------------------------
>   Line   src/Event/EventController.php
>  ------ -------------------------------------------------
>   480    Variable $reportedComment might not be defined.
>  ------ -------------------------------------------------
> 
>  ------ -------------------------------------------------
>   Line   src/Talk/TalkController.php
>  ------ -------------------------------------------------
>   447    Variable $reportedComment might not be defined.
>  ------ -------------------------------------------------
> 
>  ------ ------------------------------------------
>   Line   src/User/UserController.php
>  ------ ------------------------------------------
>   301    Variable $eventUri might not be defined.
>   471    Variable $eventUri might not be defined.
>  ------ ------------------------------------------
> 
> 
>  [ERROR] Found 4 errors

--- a/app/src/Event/EventController.php
+++ b/app/src/Event/EventController.php
@@ -458,9 +458,10 @@ class EventController extends BaseController
 
     public function reportComment($friendly_name, $comment_hash)
     {
-        $eventApi = $this->getEventApi();
-        $event    = $eventApi->getByFriendlyUrl($friendly_name);
-        $url      = $this->application->urlFor("event-comments", ['friendly_name' => $friendly_name]);
+        $eventApi        = $this->getEventApi();
+        $event           = $eventApi->getByFriendlyUrl($friendly_name);
+        $reportedComment = null;
+        $url             = $this->application->urlFor("event-comments", ['friendly_name' => $friendly_name]);
 
         $comments = $eventApi->getComments($event->getCommentsUri());
         foreach ($comments as $comment) {

--- a/app/src/Talk/TalkController.php
+++ b/app/src/Talk/TalkController.php
@@ -423,11 +423,12 @@ class TalkController extends BaseController
 
     public function reportComment($eventSlug, $talkSlug, $commentHash)
     {
-        $eventApi = $this->getEventApi();
-        $event    = $eventApi->getByFriendlyUrl($eventSlug);
-        $talkApi  = $this->getTalkApi();
-        $talk     = $talkApi->getTalkBySlug($talkSlug, $event->getUri());
-        $url      = $this->application->urlFor("talk", ['eventSlug' => $eventSlug, 'talkSlug' => $talkSlug]);
+        $eventApi        = $this->getEventApi();
+        $event           = $eventApi->getByFriendlyUrl($eventSlug);
+        $reportedComment = null;
+        $talkApi         = $this->getTalkApi();
+        $talk            = $talkApi->getTalkBySlug($talkSlug, $event->getUri());
+        $url              = $this->application->urlFor("talk", ['eventSlug' => $eventSlug, 'talkSlug' => $talkSlug]);
 
         $comments = $talkApi->getComments($talk->getCommentsUri());
         foreach ($comments as $comment) {

--- a/app/src/User/UserController.php
+++ b/app/src/User/UserController.php
@@ -245,6 +245,7 @@ class UserController extends BaseController
         $talkDb   = $this->getTalkDb();
         $talkApi  = $this->getTalkApi();
         $eventApi = $this->getEventApi();
+        $eventUri = null;
 
         $eventInfo = []; // look up an event's name and url_friendly_name from its uri
         $talkInfo  = []; // look up a talk's url_friendly_talk_title from its uri
@@ -443,6 +444,7 @@ class UserController extends BaseController
         $talkDb       = $this->getTalkDb();
         $talkApi      = $this->getTalkApi();
         $eventApi     = $this->getEventApi();
+        $eventUri     = null;
         $talkComments = $talkApi->getComments($user->getTalkCommentsUri(), true, 0);
         if (!$talkComments) {
             $this->application->redirect($this->application->urlFor('user-profile', ['username' => $username]));

